### ISSUE TITLE
`tbl-pos` is not documented

### ISSUE THREAD

**Issue by mcanouil** [quarto-dev/quarto-cli#10784]
### What would you like to do?

Report an issue on quarto.org

### Description

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/e8f7935a-19cf-4142-af9c-35f7e7b70d15">

https://quarto.org/docs/reference/formats/pdf.html#tables

**Comment by cscheid** [comment]
That's true for all crossref `*-pos` options. I think we have an open issue somewhere about the general class of problem: our automatic documentation requires fixed option names, but some names are derived programmatically, like `fig-pos`, `tbl-pos`, `lst-pos`, etc.
